### DISCUSSION TITLE
docker-compose安装的最新版本，登陆报Forbidden

### DISCUSSION THREAD

**Discussion by lijun47** [hhyo/Archery#1870] [Q&A] [answered]
![QQ图片20220923093248](https://user-images.githubusercontent.com/12150981/191877554-65648c85-6b3b-4655-8b90-eb6df51d2183.png)

**Comment by hhyo** [comment, marked as answer]
参考https://docs.djangoproject.com/en/4.0/ref/settings/#csrf-trusted-origins

**Reply by lijun47** [reply]
解决了，谢谢，.env配置CSRF_TRUSTED_ORIGINS就可以了。
